manifests/init.pp

```diff
@@ -505,6 +507,8 @@
       default: {
       }
     }
+  } elsif ($facts['os']['family'] == 'RedHat' and $facts['os']['release']['major'] == '7') {
+    require epel
```

Comment:
I don't have a strong opinion on how we should call the epel module. We could also just add it to the acceptance tests or introduce another parameter for it.